packages/workbox-range-requests/test/sw/handle-range-request.js

```diff
@@ -31,7 +31,7 @@ describe('Tests for the handle-range-request.js functions', function() {
     const reader = new FileReader();
     const finished = new Promise((resolve) => {
       reader.addEventListener('loadend', (event) => {
-        resolve(event.srcElement.result);
+        resolve(event.target.result);
```

Comment:
This isn't related to the core change in this PR, but the Firefox tests were flakey when I ran them locally before switching to `target`. Not sure why it wasn't an issue before, though...